cmake/modules/zephyr_default.cmake

```diff
@@ -31,6 +31,18 @@ find_package(ZephyrBuildConfiguration
   NO_CMAKE_SYSTEM_PACKAGE_REGISTRY
 )
 
+if(${CMAKE_VERSION} VERSION_EQUAL 3.22.2)
```

Comment:
```suggestion
# error-out if we are affected by a new CMake bug
if(${CMAKE_VERSION} VERSION_EQUAL 3.22.2 OR ${CMAKE_VERSION} VERSION_EQUAL 3.22.2)
```

Reply:
Thanks, added but removed `new`, cause `new` is not new in a year from now :wink: